x-pack/plugins/alerting/server/task_runner/transform_action_params.ts

```diff
@@ -33,6 +35,13 @@ interface TransformActionParamsOptions {
   ruleUrl?: string;
 }
 
+interface SummarizedAlertsWithAll extends SummarizedAlerts {
+  all: {
+    count: number;
+    alerts: unknown[];
+  };
```

Comment:
```suggestion
  all: {
    count: number;
    data: unknown[];
  };
```

Reply:
Done in [#3d137a1](https://github.com/elastic/kibana/pull/147360/commits/3d137a13893d66c7b29ac93e20446f4fefbba12c)